index.html

```diff
@@ -2368,7 +2351,74 @@ <h3>Status</h3>
       </section>
 
       <section>
-        <h3>Presentations</h3>
+        <h3>Verifiable Credentials</h3>
+
+        <p>
+[=Verifiable credentials=] are used to express properties about a [=subject=]
+as well as properties about the [=credential=] itself. The following properties
+are defined in this specification for a [=verifiable credential=]:
+        </p>
+
+        <dl>
+          <dt>@context</dt>
+          <dd>
+Defined in Section [[[#contexts]]].
+          </dd>
+          <dt>id</dt>
+          <dd>
+Defined in Section [[[#identifiers]]].
+          </dd>
+          <dt>type</dt>
+          <dd>
+Defined in Section [[[#types]]].
+          </dd>
+          <dt>name</dt>
+          <dd>
+Defined in Section [[[#names-and-descriptions]]].
+          </dd>
+          <dt>description</dt>
+          <dd>
+Defined in Section [[[#names-and-descriptions]]].
+          </dd>
+          <dt>issuer</dt>
+          <dd>
+Defined in Section [[[#issuer]]].
+          </dd>
+          <dt>validFrom</dt>
+          <dd>
+Defined in Section [[[#validity-period]]].
+          </dd>
+          <dt>validUntil</dt>
+          <dd>
+Defined in Section [[[#validity-period]]].
+          </dd>
+          <dt>status</dt>
+          <dd>
+Defined in Section [[[#status]]].
+          </dd>
+          <dt>credentialSchema</dt>
+          <dd>
+Defined in Section [[[#data-schemas]]].
+          </dd>
+          <dt>credentialSubject</dt>
+          <dd>
+Defined in Section [[[#credential-subject]]].
+          </dd>
+          <dt>proof</dt>
+          <dd>
+Defined in [[[VC-DATA-INTEGRITY]]].
+          </dd>
+        </dl>
+
+        <p>
+A [=verifiable credential=] can be extended to have addition properties
+through the extension mechanism defined in Section [[[#extensibility]]].
+        </p>
```

Comment:
maybe the proof property could be mentioned here instead, as an example of an additional property included via an extension mechanism?

Reply:
We link to the VC-DATA-INTEGRITY spec in the Securing Mechanisms section, so we're probably fine to just not include it in this part of the spec.

Reply:
I agree with not mentioning `proof` here.